src/UnoFlipController.java

```diff
@@ -37,8 +33,10 @@ public UnoFlipController(UnoFlipModel model){
 
     @Override
     public void actionPerformed(ActionEvent e) {
-        if (e.getActionCommand().equals(UnoFlipViewFrame.START_CMD)){
-
+        // Start game command to initialize players
+        if (e.getActionCommand().equals(UnoFlipViewFrame.START_CMD))
```

Comment:
Since we keep checking the same thing: e.getActionCommand().equals(something), we can store the return of that call in a variable and use a switch case. This is a bit more clear and easier to follow if we are checking the same variable each time. checkout this link: https://docs.oracle.com/javase/8/docs/technotes/guides/language/strings-switch.html